### PR TITLE
update cli and refactor

### PR DESCRIPTION
**Pull request by hasheemi**
PR (pull request) ini mengandung update yang ada di PR #36 dengan sedikit perubahan dan perbaikan
Isi PR ini :
- update cli dan menambahkan command run dan compile
- fixing bug di regex
- penghapusan openGroup dan closeGroup supaya menghemat baris kode :v 
- edit readme.md
- penambahan package chalk@4.1.0 dan commander@9.2.0

preview cli:
run single file
image
run multi file
image
compile singe file
image
compile multi file
image
hasil compile
image

hope you like this and thank you

### CODE REVIEW

**Comment by RioChndr** [comment]
thank you to refactor this project. please provide unit testing too, to make sure this run perfectly

**Comment by hasheemi** [comment]
How to do that ?

**Comment by hasheemi** [comment]
Oh i now , i Will do it

**Comment by hasheemi** [comment]
done

**Comment by RioChndr** [comment]
Thank you so much for your hardword, this is awesome.

File: README.md [hunk 1]
```diff
@@ -18,33 +20,48 @@ git clone https://github.com/RioChndr/jaksel-language.git
 
 ### Run
 
-If you install from npm, run jaksel script using `jaksel` command
+If you install from npm, run jaksel script using `jaksel -r <filename...>` command
+
+```bash
+jaksel -r example/example1.jaksel
```
Comment (RioChndr) [inline README.md:26]: My suggestion is make this command use old style `jaksel <file name>`

File: README.md [hunk 2]
```diff
@@ -18,33 +20,48 @@ git clone https://github.com/RioChndr/jaksel-language.git
...
+
+Or run from this repository
+
+```
+node jaksel-interpreter.js -r example/example1.jaksel
+```
+
+### Compile
+
+If you install from npm, compile jaksel script using `jaksel -c <filename...>` command
```
Comment (RioChndr) [inline README.md:37]: Please add some information about this flag

**Review by RioChndr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RioChndr) [inline README.md:26]: My suggestion is make this command use old style jaksel <file name>

**Review by RioChndr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RioChndr) [reply README.md:26]: `jaksel <file name>`

Comment (hasheemi) [reply README.md:26]: I added the -r flag for run to make it look the same as the -c flag, if the command runs only jaksel hmm it seems less consistent

Btw thanks for suggestions

File: README.md (hunk 2 above)
Comment (hasheemi) [reply README.md:37]: Okey

**Comment by hasheemi** [comment]
i add some description about -c (for compile) flag and change command to syntax

**Comment by RioChndr** [comment]
Thank you so much for refactor this project.
Sorry slow response. Saya akan review di branch khusus sebelum ke main. terima kasih banyak atas bantuan dan kerja kerasnya.

**Pull request merged**